go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/tektoncd/pipeline
 
-go 1.22
+go 1.22.3
```

Comment:
should this match the toolchain version?

Reply:
Hmm. I'm not sure 🤔. This came out of running hack/update-codegen.sh

Reply:
Fixed. Reverted this change.